packages/ui/PreviewSurvey/index.tsx

```diff
@@ -203,6 +203,14 @@ export const PreviewSurvey = ({
     }, 1000);
   };
 
+  const handleMobilePreviewModalClose = () => {
+    setIsModalOpen(false);
+    setTimeout(() => {
+      setIsModalOpen(true);
+      resetQuestionProgress();
+    }, 1000);
+  };
+
```

Comment:
This looks good, but why are we doing it differently from the desktop?

Reply:
I made it the same as on the desktop.